src/components/video/VideoCardList.vue

```diff
@@ -128,6 +128,7 @@ export default {
                 hideIgnoredTopics: true,
                 forOrg: "",
                 hidePlaceholder: false,
+                hideMissing: this.$store.state.settings.hideMissing,
```

Comment:
PTAL on this part, maybe it should be set in different way?
`this.filterConfig` below seems to be empty for channel page, effectively ignoring user settings (e.g. allowing to show placeholder streams even if user disabled them).

Reply:
hmmm. It's required that hideCollabs is false, and ignoredTopics should also be false, but i guess hidePlaceholder can still abide by the default setting.

Reply:
Makes sense. Would https://github.com/HolodexNet/Holodex/pull/671/commits/c34e5b70652f5dd352602fb2b3cda78c666b0bb0 be a better way then? I.e. respecting all settings by default and overriding only the ones that have to be forced.